samples_test/src/tests/algorithms.rs

```diff
@@ -254,6 +254,16 @@ pub const SHOR_EXPECT_DEBUG: Expect = expect![[r#"
     Found factor=17
     Found factorization 187 = 17 * 11
     (17, 11)"#]];
+pub const SIMPLEBELLPAIR_EXPECT: Expect = expect![[r#"
```

Comment:
Looking at these test cases, I'm not sure I understand why we need a regular and a debug case for each test. The two cases seem like they are the same for every test. Can we get rid of the redundancy?

Reply:
Good question. I had the same question some time ago. I believe there's a comment in the file that explains it. There're two modes of Q# interpreter and we need to test both. If we use the same expect for both Rust analyzer gets messed up.

Reply:
https://github.com/microsoft/qsharp/blob/5c7522c18ef26d3893e2f4866be7719d662cd132/samples_test/src/tests.rs#L23 - this is the place where the situation is described.